scripts/setup_truefoundry_cluster.sh

```diff
@@ -152,7 +223,7 @@ function main() {
 
     # Setup provider account if configured and cluster type is not generic
     if [ "${CLUSTER_TYPE}" != "generic" ]; then
-        setup_provider_account
+        is_cluster_provisioned
```

Comment:
this function should only be checking if the cluster is provisioned. the actual setting up of provider account should remain as it is